index.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta charset="UTF-8">
+        <title> CV </title>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+        <div class="whole-cv-container">
+            <section class="left-section">
+                <div class="photo-section">
+                    <img src="nerd.jpeg" alt="picture of face" class="profile-picture">
+                </div>
+                <div class="details">
+                    <div class="skills-and-experience line-separator">
+                        <header>
+                            <h2> skills </h2>
```

Comment:
Those spaces are not necessary.
```
<h2>skills</h2>
```
This applies to many places in your code. Please fix all of them